pushapkscript/Dockerfile

```diff
@@ -27,6 +27,9 @@ RUN python -m venv /app \
  && cd /app/configloader \
  && /app/configloader_venv/bin/pip install -r requirements/base.txt \
  && /app/configloader_venv/bin/pip install . \
- && cd /app
+ && cd /app \
+ && curl -L https://github.com/google/bundletool/releases/download/1.15.4/bundletool-all-1.15.4.jar -o /app/bundletool.jar
```

Comment:
pushflatpakscript does a similar curl download in its Dockerfile

Reply:
Sounds good! As an improvement, let's enforce the checksum here. This will remove this supply-chain attack vector. We have an example there: https://github.com/mozilla-releng/staging-mozilla-vpn-client/blob/391f3c05a766e299ee68a3ca50d645e358fdd1ad/taskcluster/docker/conda-base/Dockerfile#L8-L15

Reply:
Love it! Thank you!